src/components/App/appReducer.js

```diff
@@ -163,9 +163,10 @@ const appReducer = (state = [], action) => {
                 searchString: '',
                 mapProperties,
                 scrollTop: 0,
-                showModal: false,
```

Comment:
why showModal changed from false to true?

Reply:
The component checks if this key is false, if it's false then the modal won't show on startup